scrapy/__init__.py

```diff
@@ -28,11 +22,18 @@
 
 
 # Check minimum required Python version
-if sys.version_info < (3, 5, 2):
-    print("Scrapy %s requires Python 3.5.2" % __version__)
+if sys.version_info < (3, 6):
+    print(("Scrapy %s requires Python 3.6+" % __version__).encode())
```

Comment:
why is .encode() needed here?

Reply:
I needed to do this to get the new test to pass in Python 2.7. It has to do with the fact that we are replacing `sys.stdout` to capture. I figure, being ASCII text, that this should not be a problem. But if we can test the output without having to do this, that would definitely be better.

I got issues when using StringIO, so I switched to BytesIO. I’ll try cStringIO as mentioned in https://stackoverflow.com/a/1218951/939364, maybe that will work.

Reply:
I was concerned about this thing mostly:

```py
>>> print("foo")
foo
>>> print("foo".encode())
b'foo'
```